Roadmap.md

```diff
@@ -0,0 +1,81 @@
+
+👾 [E2E tests with sharness](https://github.com/aragon/aragon-cli/issues/358)
+
+👾 Bugfixes, see all [here](https://github.com/aragon/aragon-cli/labels/bug)
+
+- [gasLimit](https://github.com/aragon/aragon-cli/issues/346) issue
+- [gas issue on dao acl grant](https://github.com/aragon/aragon-cli/issues/350)
+- [dao new fails with ganache 2.3](https://github.com/aragon/aragon-cli/issues/321) (low priority because pinning works alright at the moment)
+
+Note: We should write tests as we fix these to ensure no regressions!
+
+👾 Small features (low hanging fruit)
+
+- remove [hard-coded gas price](https://github.com/aragon/aragon-cli/issues/353) (would be nice to let the user decide how much)
+
+👾 [Claiming an aragonid](https://github.com/aragon/aragon-cli/issues/347) 
+
+👾 Document new features (dao token commands)
+
+👾 Restructure documentation on hack.aragon to include:
+
+- Quick start / Overview / Introduction
+- Guides around specific subjects (e.g.: Publish to different environments, Using a different Ethereum account, Propagating content, etc), similar to [this](https://github.com/aragon/aragon-react-boilerplate#publish). Maybe also link more in-depth guides such as [this](https://forum.aragon.org/t/guide-custom-aragon-organization-deployment-using-the-cli/507)
+- FAQ section. We have [this](https://hack.aragon.org/docs/cli-usage.html#troubleshooting-faq) currently, but it should be it's own section and much more elaborate (If there is a recurring question on #dev-help we will add it there)
+
+## Long-term
+
+- Types (use Flow or Typescript) because it will allow us to catch more "errors" at compile time, having commands silently fail without noticing (like [this](https://github.com/aragon/aragon-cli/pull/334#discussion_r248659171)) less often. Using Typescript is also great for intellisense support (having auto-completion from IDEs) and refactoring
+- Improve smart contract tooling using new Ethereum dev tools from 0x:
+    - `[sol-trace](https://sol-trace.com)`: Human-readable stack traces
+    - `[sol-coverage](https://sol-coverage.com)`*:* Solidity code coverage
+    - `[sol-profiler](https://sol-profiler.com)`: Gas profiling for Solidity
+    - `[sol-compiler](https://sol-compiler.com)`: Solidity compilation
+
+    We plan to prioritize `sol-trace` and to look into the benefits that the others tools could have on the developer experience and decide whether to work on that *short-term*.
+
+- Identify contracts like the `MiniMeTokenFactory` which do not need to be deployed every time, but rather reused, and publish them to `aragonpm.eth` (just like the `dao-kits`)
```

Comment:
**Brett:** Ohh good idea, do we publish this every time on aragon run? We should just hardcode an address in aragen like we do with the root ENS.